Formula/licensefinder.rb

```diff
@@ -0,0 +1,37 @@
+class Licensefinder < Formula
+  desc "Find licenses for your project's dependencies"
+  homepage "https://github.com/pivotal/LicenseFinder"
+  url "https://github.com/pivotal/LicenseFinder.git",
+      tag:      "v6.14.1",
+      revision: "ced7de9f22a627cb7bd2f11f18e41ffb914ec0b0"
+  license "MIT"
+
+  depends_on "ruby@2.7" if MacOS.version <= :mojave
```

Comment:
```suggestion
  uses_from_macos "ruby", since: :mojave
```

Reply:
Shouldn't it be `since: catalina`?

Reply:
The english language can be a pain at times 😄

Reply:
@SMillerDev I'm reverting this because it will install `ruby@3` on Mojave (10.14) and LicenseFinder requires `ruby@2`.

Reply:
I think it might be better to just do `depends_on "ruby@2.7"` unconditionally then.

Reply:
> I think it might be better to just do `depends_on "ruby@2.7"` unconditionally then.

That crossed my mind but it is a kinda heavy dependency for something that would be probably used by non-ruby folks too. Also, @Rylan12 already did a similar thing for `asciidoctor` here https://github.com/Homebrew/homebrew-core/commit/e53aeb4cd25b877bbfb2b6a876e45eed842d62d4

Reply:
Yeah, it's kinda annoying. But I believe these if statements mess with the Homebrew API generation since the api is only generated once (on Linux).

So I'd like to use the full dependency or the `uses_from_macos` version.

Reply:
So, what you are suggesting is using `depends_on` with `ruby@2.7` and potentially creating another PR to the same for `asciidoctor`?

Reply:
That would be my preference, @Homebrew/maintainers other suggestions?

Reply:
It's somewhat fair to treat different macOS versions as different OS-es, in which case conditional dependency seems to make more sense to me.

Reply:
> It's somewhat fair to treat different macOS versions as different OS-es, in which case conditional dependency seems to make more sense to me.

Agreed.

Reply:
Thanks, @maxim-belkin, and @MikeMcQuaid for weighing in. 👍 
Leaving this as-is (with conditional `uses_from_macos`).